airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/JobTracker.java

```diff
@@ -127,7 +127,9 @@ public void trackSync(Job job, JobState jobState) {
       final ImmutableMap<String, Object> syncMetadata = generateSyncMetadata(connectionId);
       final ImmutableMap<String, Object> stateMetadata = generateStateMetadata(jobState);
 
-      track(MoreMaps.merge(jobMetadata, jobAttemptMetadata, sourceDefMetadata, destinationDefMetadata, syncMetadata, stateMetadata));
+      final Map<String, Object> metadata =
```

Comment:
what's the goal of this change?

Reply:
the jobnotifier used to be part of jobtracker and the metadata construction was shared but now they are distinct and I didn't clean this back to the previous state